modules/ingestor/tests/reingest/mod.rs

```diff
@@ -1,3 +1,4 @@
+mod add_sbom_group;
```

Comment:
please name it `sbom_group`. The test itself doesn't add anything, as far as I understand it.